tools/tpm2_tool.c

```diff
@@ -157,6 +158,10 @@ int main(int argc, char *argv[], char *envp[]) {
      * 'main'.
      */
     ret = tpm2_tool_onrun(sapi_context, flags) ? 1 : 0;
+    if (ret != 0) {
+            LOG_ERR("Unable to run %s: %s", argv[0], strerror(errno));
```

Comment:
errno is not guaranteed to be set correctly here by all the other tools. Plus we don't want this to be in the business of handling tool specific errors. Unable to run %s argv[0] is fine.

This could lead to the dreaded "Unable to run tpm2_foo: Success" message.

Reply:
Extra indent level on LOG_ERR